src/msg/async/PosixStack.cc

```diff
@@ -64,9 +64,13 @@ class PosixConnectedSocketImpl final : public ConnectedSocketImpl {
   }
 
   ssize_t read(char *buf, size_t len) override {
+    #ifndef _WIN32
```

Comment:
I'd suggest reversing the order (#ifdef instead of #ifndef) in this case

Reply:
Will do.